### DISCUSSION TITLE
Greetings

### DISCUSSION THREAD

**Discussion by Lanka1e** [community/community#20539] [Other Feature Feedback, Questions, & Ideas]
Hello friends 👋👋

I need a team of experienced programmers to build a start up, programmers should have good knowledge on:
* Arduino
* C programming
* python
* hardware
* security systems
* back end and front end developement
* software maintenance
* hardware maintenance
* etc

Contacte me using this WhatsApp number: +237654242311

**Comment by LJMartinez07** [comment]
Hi @Lanka1e its ok that you want to share this information here but I guess that sharing it on LinkedIn or something else similar will help you more.

**Comment by Lanka1e** [comment]
Ok thanks dear

Good day

**Comment by Developer-Bihandu** [comment]
Maybe find some people on Discord, cuz even I know some talented programmers on Discord.

**Comment by Lanka1e** [comment]
Thanks, I'll check it out.

Can you be a member of the team?? If yes then send me your WhatsApp number
with country code
